### PR TITLE
ECMS-6245 Display File name unescaped when using hexa characters

### PR DESCRIPTION
**Pull request by boubaker**
The title of a file shouldn't be treated as a JCR name knowing that the file title isn't escaped using JCR transformation anymore.

### CODE REVIEW

**Comment by thomasdelhomenie** [comment]
The file name is not ok, but when displaying the html file, the title is still unescaped:
selection_034
We should see the filename as is.

**Comment by boubaker** [comment]
done

File: core/services/src/main/java/org/exoplatform/services/cms/impl/Utils.java [hunk 1]
```diff
@@ -364,9 +364,10 @@ public static String getTitle(Node node) throws Exception {
       } else {
         title = node.getName();
       }
+      title = Text.unescapeIllegalJcrChars(title);
```
Comment (thomasdelhomenie) [inline core/services/src/main/java/org/exoplatform/services/cms/impl/Utils.java]: Shouldn't we do this in any case (not only when StringUtils.isBlank(title)) ?

Comment (boubaker) [reply core/services/src/main/java/org/exoplatform/services/cms/impl/Utils.java]: nope, we do this only for title coming from node name that was espcaed before persisted on JCR.
When storing the property exo:title, the method Text.escapeIllegalJcrChars isn't called, so we shouldn't call it when displaying the node name.

Comment (thomasdelhomenie) [reply core/services/src/main/java/org/exoplatform/services/cms/impl/Utils.java]: ok :+1:

**Pull request closed without merging**